frontend/src/metabase/lib/styling/emotion.ts

```diff
@@ -0,0 +1,5 @@
+import isPropValid from "@emotion/is-prop-valid";
+
+export function shouldNotForwardTransientProp(propName: string): boolean {
```

Comment:
I think this function name will get confusing since it is negative but is used as the value for a positive property name -- "shouldForwardProp". Can you rename this function so that it is positive as well?

Reply:
you might come up with something better, but maybe even something like `shouldForwardNonTransientProp`. The `NonTransient` part reads a little awkwardly but I think matching the positive `should*` with `should*` will help improve the clarity of the code overall.